.github/workflows/update-pages.yml

```diff
@@ -0,0 +1,31 @@
+name: Update GitHub Pages
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  update-pages:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: '14'
```

Comment:
Why did you select these version exactly? Looks like a blind copy-paste.
I just want to remind you man that our app won't work with NodeJS versions below 18.

Reply:
You  are right, the problem is that a tried to understand how does it work but for me is a something from arabic  language. I written it for in my note for asking you

Reply:
This task was on me 😄 
It's too early for you to work with this things.